net/vnstat2/Makefile

```diff
@@ -0,0 +1,62 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=vnstat2
+PKG_VERSION:=2.2
+PKG_RELEASE:=1
+PKG_MAINTAINER:=
```

Comment:
For consistency, I'd put these after PKG_BUILD_DIR. Why leave maintainer blank?

Reply:
I'd prefer it to be a community package however if a maintainer is needed I'll drop the PR unless someone wants to step up.